oaknational/Oak-Web-Application
# Fix/pupil 1053/remove worksheet answers

## Description

When downloading a lesson worksheet both the questions and the answers are returned from the api. We should not be providing the students with the answers. 

## Implementation
- swapped "worksheet-pdf" for the newly implemented "worksheet-pdf-questions" in src/components/PupilViews/PupilIntro/useWorksheetDownload.ts
- had to add to the type in src/components/SharedComponents/helpers/downloadAndShareHelpers/downloadLessonResources.tsx
this is a bit of a hack to avoid disturbing the teacher journey. We need a better solution for shared types.


## How to test

1. Go to https://deploy-preview-2935--oak-web-application.netlify.thenational.academy/pupils/lessons/transverse-waves
2. Click on intro and download
3. You should now only get the questions

## Checklist

- [x]  Check cycle one lesson download contains only worksheet questions (use lesson from above)
- [x]  Check legacy lesson download contains only worksheet questions e.g. https://deploy-preview-2935--oak-web-application.netlify.thenational.academy/pupils/programmes/computing-secondary-year-11-l/units/programming-1-sequence-2cbd/lessons/flowcharts-cct3gt/intro
